templates/carousel.hbs

```diff
@@ -0,0 +1,28 @@
+<div class="general-container">
+  <div class="carousel">
+      <div class="carousel-container">
+          <div class="card-wrapper">
+            {{#each carouselSlides}}
+              {{{module this}}}
+            {{/each}}
+          </div>
+          <div class="card-control">
+            <div class="control-container">
+              <div class="control__arrow-back arrow">
```

Comment:
Please use button instead of div here. This will make the arrow accessible using keyboard.
Styles will need to be updated.